spec/factories/etds.rb

```diff
@@ -43,6 +43,7 @@
     end
 
     factory :moomins_thesis do
```

Comment:
Do the tests pass without changes to this factory? It's not immediately clear to me what the edits to this file accomplish.